SheepDash/core/src/com/missionbit/game/states/State.java

```diff
@@ -8,10 +8,6 @@
  * Created by missionbit on 6/26/17.
  */
 
-public abstract class State {
-    protected OrthographicCamera cam;
-    protected Vector3 mouse;
-    protected GameStateManager gsm;
 
```

Comment:
Not sure why these lines were removed, but we need them. You won't be able to compile without these.